packages/front-end/components/CustomFields/CustomFieldInput.tsx

```diff
@@ -1,20 +1,21 @@
 import { FC, useEffect, useMemo, useState } from "react";
 import { CustomField, CustomFieldSection } from "back-end/types/custom-fields";
 import { UseFormReturn } from "react-hook-form";
+import { Switch } from "@radix-ui/themes";
 import { filterCustomFieldsForSectionAndProject } from "@/hooks/useCustomFields";
 import Field from "@/components/Forms/Field";
 import SelectField from "@/components/Forms/SelectField";
 import MultiSelectField from "@/components/Forms/MultiSelectField";
-import Toggle from "@/components/Forms/Toggle";
 
 const CustomFieldInput: FC<{
   customFields: CustomField[];
   // eslint-disable-next-line
   form: UseFormReturn<any>;
   section: CustomFieldSection;
+  setCustomFields?: (customFields: Record<string, string>) => void;
```

Comment:
We should not have both this and `form` as props.  I would recommend removing `form` entirely and instead having this component take `value` and `setValue` props.  Will have better type safety and make this component more flexible.

Reply:
done